src/components/Table/components/TableHead/tableHead.tsx

```diff
@@ -1,62 +1,55 @@
+import SouthRoundedIcon from "@mui/icons-material/SouthRounded";
 import {
   TableHead as MTableHead,
   TableRow as MTableRow,
   TableCell,
   TableSortLabel,
 } from "@mui/material";
-import { flexRender, RowData, Table } from "@tanstack/react-table";
+import { flexRender, RowData } from "@tanstack/react-table";
 import React, { Fragment } from "react";
 import { ROW_DIRECTION } from "../../common/entities";
-import { getTableSortLabelProps } from "../../common/utils";
 import {
   getTableCellAlign,
   getTableCellPadding,
 } from "../TableCell/common/utils";
-
-export interface TableHeadProps<T extends RowData> {
-  rowDirection: ROW_DIRECTION;
-  tableInstance: Table<T>;
-}
+import { handleToggleSorting } from "../TableFeatures/RowSorting/utils";
+import { TableHeadProps } from "./types";
+import { isTableSortLabelDisabled } from "./utils";
 
 export const TableHead = <T extends RowData>({
   rowDirection,
   tableInstance,
 }: TableHeadProps<T>): JSX.Element => {
+  const {
+    options: { enableSortingInteraction },
```

Comment:
`enableSortingInteraction` is now on table options.

Reply:
`enableSortingInteraction` allows the table to be sortable, but controls whether or not the user can interact with sorting i.e. view sorting on a column, or interact with table sorting.